ui/plugin-system/src/components/PluginEditor/plugin-editor-api.ts

```diff
@@ -104,11 +122,19 @@ export function usePluginEditor(props: UsePluginEditorProps) {
         kind: nextKind,
         spec: previousState,
       });
-      return;
+      // return;
```

Comment:
Is this `return` intentionally commented out?

Reply:
yes! i should probably delete this because now it needs to continue and check if it should trigger `onHideQuery`